### ISSUE TITLE
Error: bootConfig() must be protected

### ISSUE THREAD

**Issue by jeremydouglas** [silentzco/statamic-charge#9]
I'm getting this error when loading any view after upgrading to `4.0.0-rc.3` and Statamic 3:

```
Access level to Silentz\Charge\ServiceProvider::bootConfig() must be protected (as in class Statamic\Providers\AddonServiceProvider) or weaker
```

**Comment by edalzell** [comment]
Yup, have to change all my add ons with this change. Getting there, sorry.

**Comment by edalzell** [comment]
Alright, think this is resolved in 4.0.1, just released.

Please re-open if still an issue.
